aptos-move/framework/aptos-framework/sources/account/auth_data.move

```diff
@@ -1,18 +1,40 @@
 module aptos_framework::auth_data {
+    use std::string::String;
+
+    enum DomainAccount has copy, drop {
+        V1 {
+            domain_name: String,
+            account_authentication_key: vector<u8>,
```

Comment:
What's this? @igor-aptos

Reply:
this is the identifier of the source account.

i.e. in what David wrote above - for the account address account will have - sha3_256(FI::data::5), that is the data.

It can be an address of the original chain for example, in the format implementation of AA function will use

Reply:
Then it seems inappropriate to let user decide this field. Maybe it should be derived.

Reply:
I meant - the implementation of AA function needs to verify that this matches with the signature. 

I am not sure what "derived" means?

Reply:
ignore my "derived" words.
> the implementation of AA function needs to verify that this matches with the signature.
Then for a key pair, this should be the public key part which is part of the `authenticator` itself.

Reply:
>  Then for a key pair, this should be the public key part which is part of the authenticator itself.

yes, but aa dispatch cannot look where in authenticator is, right? so it seems like it needs to be here, and implementation of AA needs to check that it matches what is in the authenticator?

Reply:
oh yeah, the implemetation(AA function info) actually knows, and it can represent the domain itself.